app/commands/user_track/generate_concepts_status.rb

```diff
@@ -0,0 +1,27 @@
+class UserTrack
+  class GenerateConceptsStatus
+    include Mandate
+
+    initialize_with :user_track
+
+    def call
+      concepts_status
+    end
+
+    private
+    memoize
+    def concepts_status
```

Comment:
```suggestion
```

Reply:
I think you are suggesting to merge the private method into the call method, but line 8 is missing from your suggestion

Reply:
I was. Sorry :)